File: DSA/Hritvik_500120426/Day1/Day1Question3.java
import java.util.Scanner;

public class Day1Question3 {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter size of array:");
        int size=sc.nextInt();
        int[] arr=new int[size];
        int I,sum=0;
        for( I=0; I<size;i++){
            System.out.println("Enter "+(I+1)+ " Element:");
            arr[I]=sc.nextInt();
            sum=sum+arr[I];
        }
        for (I=0;i<size;i++){
            System.out.print(arr[I]+(I<size-1?",":"\n"));
        }
        System.out.println("Sum of Elements of array:"+sum);
    }
}

File: DSA/Nivedan_590014904/Day_1_Question_3.c
#include<stdio.h>
void findsum(int arr[],int n){
int i=0 , sum=0;
for(i=0; i<n ;i++){
sum = arr[i] + sum;
}
printf("The Sum of the array is %d",sum);
}
int main(){
    int n;
    printf("Enter the Length of the Array: ");
    scanf("%d",&n);
    printf("Enter %d elements:\n", n);
     int arr[n];  
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }
 findsum(arr,n);
}
